apps/aklite-offline/cmds.cpp

```diff
@@ -12,6 +12,16 @@ int InstallCmd::installUpdate(const Config& cfg_in, const boost::filesystem::pat
         offline::client::install(cfg_in, {src_dir / "tuf", src_dir / "ostree_repo", src_dir / "apps"});
 
     switch (install_res) {
+      case offline::PostInstallAction::NeedRebootForBootFw: {
+        ret_code = 90;
+        std::cout
+            << "Please reboot a device to confirm a boot firmware update, and then run the `install` command again\n";
+        std::cout << "If the reboot doesn't help to proceed with the update, then make sure that "
+                     "`bootupgrade_available` is set to `0`.";
+        std::cout << "Try running `fw_setenv|fiovb_printenv bootupgrade_available 0`, reboot a device, and then run "
```

Comment:
should be fiovb_setenv bootupgrade_available 0

Reply:
Yep, it should be `fiovb_setenv`